internal/caveats/errors.go

```diff
@@ -0,0 +1,107 @@
+package caveats
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/rs/zerolog"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+
+	"github.com/authzed/spicedb/pkg/caveats"
+	core "github.com/authzed/spicedb/pkg/proto/core/v1"
+	"github.com/authzed/spicedb/pkg/spiceerrors"
+)
+
+// CaveatEvaluationErr is an error in evaluation of a caveat expression.
+type CaveatEvaluationErr struct {
```

Comment:
given the package name is `caveat`, the `Caveat` prefix in the error types are redundant.

`caveat.EvaluationErr` better than `caveat.CaveatEvaluationErr`

Reply:
Done